components/app_header/Container.vue

```diff
@@ -15,7 +15,7 @@
       </div>
 
       <div class="ml-3 flex gap-10 text-bcgov-blue5">
-        <AppHeaderNavLink text="Admin" :route="Route.Admin" />
+        <AppHeaderNavLink text="Admin" :route=adminURL />
```

Comment:
Make the link open in a new tab
```suggestion
        <AppHeaderNavLink text="Admin" :route=adminURL target="_blank"/>
```